flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/source/fetch/PostgresScanFetchTask.java

```diff
@@ -394,6 +423,46 @@ private void createDataEventsForTable(
             }
         }
 
+        /**
+         * Create a slot before snapshot reading so that the slot can track the WAL log during the
+         * snapshot reading phase.
+         */
+        private void createSlotForBackFillReadTask() {
+            try {
+                SlotState slotInfo = null;
+                try {
+                    slotInfo = jdbcConnection.getReplicationSlotState(slotName, pluginName);
+                } catch (SQLException e) {
+                    LOG.warn(
+                            "unable to load info of replication slot, will try to create the slot");
+                }
+                if (slotInfo == null) {
+                    try {
+                        replicationConnection.createReplicationSlot().orElse(null);
+                    } catch (SQLException ex) {
+                        String message = "Creation of replication slot failed";
+                        if (ex.getMessage().contains("already exists")) {
+                            message +=
+                                    "; when setting up multiple connectors for the same database host, please make sure to use a distinct replication slot name for each.";
+                        }
+                        throw new FlinkRuntimeException(message, ex);
+                    }
+                }
+                waitForReplicationSlotReady(120, jdbcConnection, slotName, pluginName);
```

Comment:
120 * 2s is too long for me, do you refer any materials about the retry times `120`?
could we decrease to 30 which will timeout after 1 minute?